Celeste.Mod.mm/MonoModRules.cs

```diff
@@ -2408,5 +2411,29 @@ bgt.un.s	ldloca.s
                 PostProcessType(modder, nested);
         }
 
+        public static void PatchEntityListUpdate(ILContext context, CustomAttribute attrib) {
+            TypeDefinition Entity = MonoModRule.Modder.FindType("Monocle.Entity").Resolve();
+            MethodDefinition entity_UpdatePreceder = Entity.FindMethod("UpdatePreceder");
+            MethodDefinition entity_UpdateFinalizer = Entity.FindMethod("UpdateFinalizer");
+
+            ILCursor cursor = new ILCursor(context);
+            ILLabel activeBranchLabel = cursor.DefineLabel();
+            ILLabel loopBranchLabel = null;
+            if (!cursor.TryGotoNext(MoveType.Before, instr=>instr.MatchBr(out loopBranchLabel))) {
+                throw new Exception("Label not found. This is a temporary error that shouldn't be shown to you unless you know what you're doing.");
+            }
+            ILCursor clone = cursor.Clone();
+            clone.GotoNext(MoveType.After, instr => instr.MatchLdloc(1), instr => instr.MatchCallvirt("Monocle.Entity", "Update"));
+            clone.MarkLabel(activeBranchLabel);
+            cursor.GotoNext(MoveType.After, instr => instr.MatchStloc(1));
+            cursor.Emit(OpCodes.Ldloc_1);
+            cursor.Emit(OpCodes.Callvirt, entity_UpdatePreceder);
+            cursor.GotoNext(MoveType.After, instr => instr.MatchLdloc(1), instr => instr.MatchLdfld("Monocle.Entity", "Active"));
+            cursor.Remove();
+            cursor.Emit(OpCodes.Brfalse, activeBranchLabel);
```

Comment:
`cursor` and `clone` are confusing variable names without any context. Please rename, or document the purpose of the clone with a comment so that it can be identified better at a quick glance.

It's apparently used to mark a label, which can be done in one step without the pre-definition of `activeBranchLabel`:
![image](https://user-images.githubusercontent.com/1200380/150684358-cae0b876-fe87-46cf-a979-cfab6defe465.png)

Given how it's used, it makes me wonder if juggling the clone around is necessary to begin with.